healthcare.management.system/rest-module/src/main/java/com/healthcare/management/system/rest/controller/UserController.java

```diff
@@ -0,0 +1,64 @@
+package com.healthcare.management.system.rest.controller;
+
+import com.healthcare.management.system.rest.entity.User;
+import com.healthcare.management.system.rest.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+// TODO: 18.08.22 add version in the api endpoint path
+
+public class UserController {
+    private UserService userService;
+
+    @Autowired
+    public  UserController(UserService theUserService) {
+        userService = theUserService;
+    }
+
+    @GetMapping("/users")
+    public List<User> findAllUsers(){
+        return userService.findAll();
+    }
+
+    @GetMapping("/users/{userId}")
+    public User getUser(@PathVariable int userId){
+        User theUser = userService.findById(userId);
+
+        if (theUser == null) {
+            throw new RuntimeException("There is no user with id: " + userId);
```

Comment:
Maybe you can create a dedicated `UserNotFoundException` which extends `RuntimeException`, e.g.:
```
public UserNotFoundException(int userId) {
   super("There is no user with id: " + userId);
}
```

This will be helpful when defining response codes for exceptions